controllers/apply/awards-for-all/fields.js

```diff
@@ -1780,6 +1919,13 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                         en: `Select the amount of people who speak Welsh that will benefit from your project`,
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `The beneficiaries must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
I'm not sure what field this is for - @NikhilNanjappa can you please clarify?

Reply:
It's a list of people who'll speak Welsh (half, less than half, etc) which probably doesn't need this validation as it's not free text...?

Reply:
Thanks Matt - that's what I was thinking too.

Reply:
My bad .. I will remove this validation from here.